ci/restore-vcpkg-from-cache.sh

```diff
@@ -27,7 +27,7 @@ if [[ -x "${HOME}/.cache/bin/vcpkg" ]]; then
 else
   (cd "${VCPKG_ROOT}"  && ./bootstrap-vcpkg.sh -useSystemBinaries)
```

Comment:
FWIW, I believe `"${VCPKG_ROOT}"/bootstrap-vcpkg.sh` works, regardless of your CWD.